codespell_lib/data/dictionary.txt

```diff
@@ -14924,6 +14927,7 @@ mutches->matches
 muti->multi
 muticast->multicast
 mutices->mutexes
+mutiindex->multiindex, multi index,
```

Comment:
`Multiindex` should be hyphenated unless it's a special word in a programming language, in which case it probably shouldn't be first.

Reply:
b9992e9